src/module-config/module-config.test.ts

```diff
@@ -308,15 +308,47 @@ describe("getConfig", () => {
   });
 });
 
+it("fills array element object elements with defaults", async () => {
+  Config.defineConfigSchema("foo-module", {
+    foo: {
+      default: [{ a: { b: "arrayDefaultB", filler: "arrayDefault" } }],
+      arrayElements: {
+        a: {
+          b: { validators: [] },
+          filler: { default: "defaultFiller", validators: [isString] }
```

Comment:
The bug in the previous attempt stemmed from the absence of validators in the test. So the code to fill defaults was mishandling validators.